incubator/cloudbeaver/files/nuodb-config.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+#This script is performing the steps here https://github.com/dbeaver/cloudbeaver/wiki/Adding-new-database-drivers#adding-drivers-in-cloudbeaver
+#It may break with future releases of cloudbeaver, but the benefit is that we don't need to mantain our own version of the cloudbeaver image for use with NuoDB.
+
+#Get the driver filename to fetch, or default to the current version at time of writing
+NUODB_DRIVER_VERSION="${NUODB_DRIVER_VERSION:=nuodb-jdbc-23.0.0.jar}";
+NUODB_DRIVER_NAME="nuodb-jdbc-$NUODB_DRIVER_VERSION.jar";
+
+rm -rf /opt/cloudbeaver/drivers/nuodb/*;
+if [ "$DOWNLOAD_DRIVER" = "true" ]
+then
+  #Download the driver from maven and place it in the cloudbeaver driver directory
+  echo "Downloading NuoDB Driver $NUODB_DRIVER_NAME from maven";
+  wget -O /opt/cloudbeaver/drivers/nuodb/$NUODB_DRIVER_NAME https://repo1.maven.org/maven2/com/nuodb/jdbc/nuodb-jdbc/$NUODB_DRIVER_VERSION/$NUODB_DRIVER_NAME;
+else
+  #Decode it from the configmap
+  echo "Copying NuoDB Driver $NUODB_DRIVER_NAME from configmap";
+  cp /opt/cloudbeaver/config/nuodb/$NUODB_DRIVER_NAME /opt/cloudbeaver/drivers/nuodb/$NUODB_DRIVER_NAME;
+fi;
+
+#Replace the two plugin.xml files containing the additional NuoDB config
+CLOUDBEAVER_BUNDLE_NAME=$(find /opt/cloudbeaver/cloudbeaver-jars/ -name "io.cloudbeaver.resources.drivers.base*");
```

Comment:
Should the script fail if it doesn't find the bundle?

Reply:
Added check/fail for no find result